submissions/JuliaVorotchenko/Popup-task/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+              content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="main.css">
+        <link href="https://fonts.googleapis.com/css?family=Work+Sans:400,500,600,700" rel="stylesheet">
+
+        <title>HTML & CSS practice: Google-style Popup</title>
+</head>
+<body>
+    <!--Меню-->
+    <div class="nav-wrapper">
+        <div class="navigation">
+            <div class="nav-item text"><a href="#">Gmail</a></div>
+            <div class="nav-item text"><a href="#">Images</a></div>
+            <!--Popup-->
+            <div class="nav-item nav-item-icon grid"><a href="#" class="grid-link"><img src="img/grid.svg" alt="grid menu icon"></a>
+                <input class="grid-top" type="checkbox" id="applications" name="" value="">
```

Comment:
Better if Checkbox has the same size as `grid` has
http://i.imgur.com/1npBYK1.png